generators/server/templates/src/main/java/package/web/rest/_AccountResource.java

```diff
@@ -200,17 +201,20 @@ public UserDTO getAccount(Principal principal) {
                 return userDTO;
             } else {
                 // Allow Spring Security Test to be used to mock users in the database
-                return Optional.ofNullable(userService.getUserWithAuthorities())
-                    .map(UserDTO::new)
-                    .orElseThrow(RuntimeException::new);
+                Optional<User> user = Optional.ofNullable(userService.getUserWithAuthorities());
+                if (user.isPresent()) {
+                    return new UserDTO(user.get());
+                } else {
+                    throw new DefaultException("User could not be found");
+                }
             }
         } else {
-            throw new RuntimeException();
+            throw new DefaultException("User could not be found");
```

Comment:
`throw ProblemBuilder.withTitle("User could not be found").build());`

Reply:
Unfortunately those doesn't work, as `withTitle` is not a static method

Reply:
Oups ! It's `Problem.builder().withTitle...` !